cli/utils/common.go

```diff
@@ -0,0 +1,15 @@
+package utils
+
+import (
+	"io"
+	"strings"
+)
+
+func StringToIOReader(s string) io.Reader {
+	return strings.NewReader(s)
+}
+
+func IOReadCloserToString(r io.ReadCloser) string {
+	b, _ := io.ReadAll(r)
+	return string(b)
+}
```

Comment:
don't you have to close the reader?

```suggestion
func IOReadCloserToString(r io.ReadCloser) string {
    defer r.Close()
	b, _ := io.ReadAll(r)
	return string(b)
}
```